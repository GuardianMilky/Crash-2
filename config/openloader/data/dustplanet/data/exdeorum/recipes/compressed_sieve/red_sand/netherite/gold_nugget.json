{"type":"exdeorum:compressed_sieve","ingredient":{"tag":"exdeorum:compressed/red_sand"},"mesh":"exdeorum:netherite_mesh","result":"minecraft:gold_nugget","result_amount":{"type":"minecraft:binomial","n":7.0,"p":0.15}}