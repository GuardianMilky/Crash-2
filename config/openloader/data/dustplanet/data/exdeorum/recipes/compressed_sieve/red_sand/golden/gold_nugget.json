{"type":"exdeorum:compressed_sieve","ingredient":{"tag":"exdeorum:compressed/red_sand"},"mesh":"exdeorum:golden_mesh","result":"minecraft:gold_nugget","result_amount":{"type":"minecraft:binomial","n":7.0,"p":0.19}}